apps/selvbetjening-ui/src/inntektsjustering/3-oppsummering/InntektsjusteringOppsummering.tsx

```diff
@@ -50,18 +52,41 @@ export const InntektsjusteringOppsummering = () => {
                                     <FormSummary.Label>Hvor mye har du oppgitt i arbeidsinntekt?</FormSummary.Label>
                                     <FormSummary.Value>{inntekt.arbeidsinntekt} kr</FormSummary.Value>
                                 </FormSummary.Answer>
+
                                 <FormSummary.Answer>
                                     <FormSummary.Label>Hvor mye har du oppgitt i næringsinntekt?</FormSummary.Label>
                                     <FormSummary.Value>{inntekt.naeringsinntekt} kr</FormSummary.Value>
                                 </FormSummary.Answer>
+                                {/* TODO: bruker hardkodet person helt til vi har dette på plass i backend */}
+                                {finnAlder({ foedselsdato: new Date(1968, 4, 11) }) === Alder.FEMTI_SYV_TIL_SEKSTI && (
+                                    <>
+                                        <FormSummary.Answer>
+                                            <FormSummary.Label>Hva har du oppgitt i AFP inntekt?</FormSummary.Label>
```

Comment:
Denne burde vært `innhold?.inntektSkjemaer?.femtiSyvTilSekstiAarSkjema?.AFPTjenestepensjonordning?.label?.[spraak]` ?

Gjelder alle `Labels`

Reply:
Designed er ikke 100% på plass, så har ventet med sanity schemaene for oppsummeringen